cardano-crypto-class/src/Cardano/Crypto/Seed2.hs

```diff
@@ -0,0 +1,145 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+
+-- | Seeds for key generation.
+--
+-- Work in progress, Seeds backed up by secure memory.
+--
+module Cardano.Crypto.Seed2
```

Comment:
Maybe leave this module out of this PR, if it's still WIP?

Reply:
This is actually a leftover. Will removed.

Reply:
removed